content/en/apps/reference/api.md

```diff
@@ -818,14 +818,27 @@ Content-Type: application/json; charset=utf-8
 
 ### POST /api/v1/users
 
-Create a new user with a place and a contact.
+Create new users with a place and a contact.
+
+ Creating multiple users at once by passing an array of users was introduced in version 3.15.  
+All users need to meet the following requirements before any of them are created:
+- All required fields are filled in
```

Comment:
Are the required fields listed somewhere?

Reply:
Yes, they're mentioned just above, at the beginning of the `Users` section  
https://docs.communityhealthtoolkit.org/apps/reference/api/#supported-properties-2

Reply:
Awesome!